pkg/names/names.go

```diff
@@ -0,0 +1,15 @@
+package names
```

Comment:
could you introduce this change in a different commit please?

Reply:
done